rest_framework_elasticsearch/es_mixins.py

```diff
@@ -67,7 +71,7 @@ def es_paginator(self):
         The es_paginator instance associated with the view
         """
         if not hasattr(self, '_es_paginator'):
-            if self.es_paginator_class is None:
+            if self.es_pagination_class is None:
                 self._es_paginator = None
             else:
                 self._es_paginator = self.es_paginator_class()
```

Comment:
You will get an error in this line, please update to:
self._es_paginator = self.es_pagination_class()